components/brave_new_tab_ui/containers/newTab/index.tsx

```diff
@@ -577,6 +593,30 @@ class NewTabPage extends React.Component<Props, State> {
     )
   }
 
+  renderBraveVPNWidget (showContent: boolean, position: number) {
```

Comment:
optional (to avoid needing a bind when calling):


```suggestion
  renderBraveVPNWidget = (showContent: boolean, position: number) => {
```

Reply:
Fixed.